### DISCUSSION TITLE
How do i get --fix and vscode quick fixes for graphql in code files?

### DISCUSSION THREAD

**Discussion by jthoward64** [graphql-hive/graphql-eslint#2799] [Q&A] [answered]
Is it even possible? All of our graphql is inline in ts/tsx files and ESLint does not show anything as auto fixable.

**Comment by dimaMachina** [comment, marked as answer]
fixes / suggestions are supported for code files, see on rules table if rule supports fixes/suggestions https://the-guild.dev/graphql/eslint/rules

**Reply by jthoward64** [reply]
Any idea why I might not be seeing them?
